### PR TITLE
Latest CustomAsteroids configs need recent CA version.

### PR DESCRIPTION
**Pull request by Starstrider42**
This PR updates versions 1.6+ of stock Custom Asteroids configs to depend on Custom Asteroids 1.6 or later. Previous versions of the configs may also be used with CA 1.6, but new configs may not work with older versions of Custom Asteroids.

### CODE REVIEW

**Comment by Starstrider42** [comment]
I think the build is broken on the Custom Asteroids 1.6 entry not yet being generated... I assume that will happen with time.

**Comment by HebaruSan** [comment]
Yeah, updates can take up to 3-4 hours after versions are uploaded thanks to KSP-SpaceDock/SpaceDock#192.

**Comment by HebaruSan** [comment]
#rebuild

File: NetKAN/CustomAsteroids-Pops-Stock-Inner.netkan [hunk 1]
```diff
@@ -45,7 +45,7 @@
         {
             "version" : [ ">= v1.6.0", "< v2.0.0" ],
             "override" : {
-                "comment": "Match to Custom Asteroids v1.6.0, see NetKAN#3841.",
+                "comment": "Match to Custom Asteroids v1.6.0, see NetKAN#6963.",
```
[the remark this answers is not in the document]
Comment (HebaruSan) [reply NetKAN/CustomAsteroids-Pops-Stock-Inner.netkan]: OK, I'll change it back.
Also I feel like I finally understand what KSP-CKAN/CKAN#1693 is talking about!

**Pull request merged**